docs/model-deployment-prepackaged-server-pytorch.md

```diff
@@ -0,0 +1,61 @@
+```
+
+The result is:
+```bash
+{"data":{"names":[],"ndarray":[[-1.3545889854431152,-2.0941741466522217,0.39545008540153503,3.433469295501709,-0.08178478479385376,1.7413524389266968,0.9827514290809631,-1.3434531688690186,0.6330257654190063,-1.5354186296463013]]},"meta":{}}
+```
+
+## How It Works
+
+You can check the detailed code in the [Github](https://github.com/InfuseAI/primehub-seldon-servers/tree/master/pytorch).
+Here, we demonstrate the pseudo code for you in order to easily understand.
+
+We assume PyTorch models saved using state_dict method. The model file is saved as `xxx.pt`. 
+The model class file is saved as `xxx.py` and the model class is named as `PyTorchModel`. 
+The model file and the model class file must be in the same directory and there must be only one model file and one model class file.
+
+Therefore, when we intialize the PyTorch model:
+```python
+def __init__(self, model_uri):
+    model_file, model_class_file = load_from_model_uri(model_uri)
+    model_class = load_model_class(model_class_file)
+    self.model = model_class()
+    self.model.load_state_dict(torch.load(model_file))
+    self.model.eval()
+```
+
+And, when a client send the request, it will call the predict function:
+```python
+def predict(self, X):
+    with torch.no_grad():
+        inputs = torch.from_numpy(X).float()
+        return self.model(inputs).tolist()
```

Comment:
remove float() and change toList() to numpy()

Reply:
Fixed. Thx!